extensions/eo/README.md

```diff
@@ -87,59 +105,58 @@ Asset definitions that contain band data should reference the band index. Each a
 | ---------- | --------- | -------------------------------------------- |
 | eo:bands   | \[number] | Lists the band names available in the asset. |
 
-See [example-landsat8.json](examples/example-landsat8.json) for a full example.
+See the [landsat8 Item example](examples/landsat8-item.json) for a full example.
 
 ```js
```

Comment:
This example is strange. Is that meant to be a collection? If yes, why's there the Item view extension mentioned?